README.rst

```diff
@@ -52,6 +52,12 @@ To use edx-sphinx-theme for a repository's documentation:
 * Use the ``AUTHOR`` constant where appropriate in conf.py
   (this default is only provided as a convenience; the repository is free
   to use another value if appropriate).
+* If you would like to add an "Edit on Github" link to every page, add a dict called html_context with the following keys
+  - display_github: True or False # Bool to set whether to integrate with Github
+  - github_user: edx # username or orgname
+  - github_repo: # repositories name
+  - github_version: "maser" # The main/master branch of repository
+  - conf_py_path: "/docs/" # Path in checkoot to root of the docs directory. At edx, by convention, it is /docs/
```

Comment:
Wordy and had typos. I think it is simpler to just use the example code. Thoughts?
```suggestion
* If you would like to add an "Edit on Github" link to every page, add a dict called ``html_context``, as detailed in the following example.
```

Reply:
I agree. I